freeipa/src/main/java/com/sequenceiq/freeipa/controller/UserV1Controller.java

```diff
@@ -102,13 +113,22 @@ public SyncOperationStatus setPassword(SetPasswordRequest request) {
     @Override
     @CheckPermissionByAccount(action = AuthorizationResourceAction.ENVIRONMENT_WRITE)
     public SyncOperationStatus getSyncOperationStatus(@NotNull String operationId) {
-        checkUserCrn();
+        checkActorCrn();
         String accountId = ThreadBasedUserCrnProvider.getAccountId();
         LOGGER.debug("getSyncOperationStatus() requested for operation '{}' in account '{}'", operationId, accountId);
         return operationToSyncOperationStatus.convert(
                 operationService.getOperationForAccountIdAndOperationId(accountId, operationId));
     }
 
+    @Override
+    @CheckPermissionByResourceCrn(action = AuthorizationResourceAction.ENVIRONMENT_READ)
```

Comment:
I think you should use `@CheckPermissionByAccount` here (similarly to other methods), now every permission check on API methods happens on account level in freeipa, internal calls ( with `@TenantAwareParam`) still will work, resource based authz implementation is in progress,

Reply:
done